unit-tests/py/test.py

```diff
@@ -72,7 +72,10 @@ def print_stack():
     """
     Function for printing the current call stack. Used when an assertion fails
     """
+    test_py_path = "librealsense" + os.sep + "unit-tests" + os.sep + "py" + os.sep + "test.py"
```

Comment:
This assumes the name of the repo is `librealsense` but it may not be (mine is `lrs`)

Reply:
you are correct, I remove that part. Should still work without it